Resources/doc/index.rst

```diff
@@ -122,7 +122,7 @@ start:
 
         namespace AppBundle\Entity;
 
-        use FOS\UserBundle\Entity\User as BaseUser;
+        use FOS\UserBundle\Model\User as BaseUser;
```

Comment:
In FOSUserBundle 1.3 (for which you are proposing this change), you would have extended the entity class. For FOSUserBundle 2.x, the documentation was already updated to reflect the changed structure.